data/boarding missions.txt

```diff
@@ -8,32 +8,335 @@
+		decline
+
+mission "Boarding Pirate (Small)"
+	boarding
+	repeat
+	minor
+	to offer
+		random < 7
+	source
+		government "Pirate"
+		category "Interceptor" "Light Warship" "Transport" "Light Freighter"
+	on offer
+		payment 50000
+		conversation "boarding pirate"
+	destination "Earth"
+
+mission "Boarding Pirate (Medium)"
+	boarding
+	repeat
+	minor
+	to offer
+		random < 5
+	source
+		government "Pirate"
+		category "Medium Warship" "Heavy Freighter"
+	on offer
+		payment 100000
+		conversation "boarding pirate"
+	destination "Earth"
+
+mission "Boarding Pirate (Large)"
+	boarding
+	repeat
+	minor
+	to offer
+		random < 3
+	source
+		government "Pirate"
+		category "Heavy Warship"
+	on offer
+		payment 150000
+		conversation "boarding pirate"
+	destination "Earth"
+
+
+
+mission "Pirate Ambush"
+	invisible
+	boarding
+	repeat
+	minor
+	deadline 1
+	to offer
+		"combat rating" > 100
+		random < 5
+	source
+		government "Pirate"
+		category "Interceptor" "Light Warship" "Transport" "Light Freighter"
+	on offer
+		conversation
+			`You board the <origin>, but find no crew aboard. When you reach the cockpit of the ship, you look out into space to see an escape pod floating away. The <origin> is emitting a distress signal to nearby pirates, and a self destruct sequence has been activated. It looks like you're about to have company.`
+				launch
+	
+	npc kill
+		government "Pirate"
+		personality nemisis harvests plunders entering
+		fleet "pirate raid" 2
+
+
+
+mission "Pirate Mutiny"
+	boarding
+	minor
+	description "Bring <bunks> ex-pirate crew members to <destination>."
+	to offer
+		random < 10
+	source
+		government "Pirate"
+		category "Medium Warship" "Heavy Warship"
+	destination
+		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
+		distance 2 4
+	passengers 10 20
+	on offer
+		conversation
+			`When you board the <origin>, you find most of the crew already dead. One of the surviving crew members, who looks like a teenaged boy no older than 16, approaches you with his hands in the air. He explains that after the ship was disabled, a section of the crew decided to overthrow the tyrannous captain. He continues, saying how most pirate crew members are forced into servitude on pirate ships. The surviving <bunks> crew members beg you to bring them to <destination> so that they can escape from their lives of piracy.`
+			`	The fight seems to have damaged the ship beyond repair. This ship may not last much longer in the vacuum of space if you leave the pirates here.`
+			choice
+				`	(Help the pirate crew.)`
+					launch
+				`	(Leave the pirate crew.)`
+					flee
+	on complete
+		payment 25000
+		dialog `The pirate crew members thank you for bringing them to <planet>. One of them hands you a small payment of <payment>. You wish them all the best of luck with their new lives and part ways.`
```

Comment:
Sidenote: that's not a really a "small payment", isn't it? Consider these are probably impoverished teens forced into labor. For scale, it took Joe & Maria 1 year to get together 35000 Credits.

Reply:
This is 10-20 so the cost between them is between 2500 and 1250, between 1 and 2 months for an (averaged out) Joe/Maria. Although it might be reasonable to assume they are living hand-to-mouth I would say this is in the realms of possibility.

Reply:
And they did just kill a pirate captain. Among the surviving crew, I'm sure someone was like "hold on, let me pick his body clean of any valuables before we leave" and found enough credits to pay you. (And if they did that, they're probably keeping most of it.)